src/integration/billing/BillingIntegration.ts

```diff
@@ -247,18 +248,23 @@ export default abstract class BillingIntegration {
         action: ServerAction.BILLING_TRANSACTION
       });
     }
-  }
-
-  private async _getUsersWithNoBillingData(): Promise<User[]> {
-    const newUsers = await UserStorage.getUsers(this.tenant,
-      {
-        statuses: [UserStatus.ACTIVE],
-        notSynchronizedBillingData: true
-      }, Constants.DB_PARAMS_MAX_LIMIT);
-    if (newUsers.count > 0) {
-      return newUsers.result;
+    if (!chargingStation) {
+      throw new BackendError({
+        message: 'The charging station is mandatory to start a transaction',
+        module: MODULE_NAME,
+        method: 'checkStartTransaction',
+        action: ServerAction.BILLING_TRANSACTION
+      });
+    }
+    // Check for the Site Area
+    if (!chargingStation.siteArea) {
```

Comment:
If Org is not active, no Site Area will be provide.
Check if the org component is active.
Missing unit tests? ;-)